lib/wizard.js

```diff
@@ -60,23 +70,20 @@ const Wizard = (steps, fields, settings) => {
 
     options.i18n = settings.i18n;
 
-    const SuperClass = options.controller || settings.controller;
-
     if (options.controller) {
       deprecate(
         'hof-form-wizard: Passing a custom step controller is deprecated',
         'Instead give one or more behaviours which will be mixed in to the base controller'
       );
     }
 
-    const behaviours = options.behaviours ? _.castArray(options.behaviours) : [];
-
-    const controller = new (behaviours.length ?
-      createController(SuperClass, behaviours) :
-      SuperClass)(options);
+    const SuperClass = options.controller || settings.controller;
+    const Controller = getController(SuperClass, options.behaviours);
+    const controller = new Controller(options);
 
     controller.use([
       require('./middleware/check-session')(route, controller, steps, first),
+      require('./middleware/check-complete')(route, controller, steps, first),
```

Comment:
If you hit `/` it will reset the session and redirect to itself. The once redirected it will render normally because the `complete` flag won't exist on the session anymore.

In terms of mimicking the existing start controller functionality, that's what this whole PR does, except it doesn't apply specifically to one step - all steps will restart the session once it is flagged complete. It removes the need to have a special empty step at the beginning of an app.